test/files/not-pseudo-selector.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8">
+<title>:not pseudo selector</title>
+<style>
+    .foo div {
+       width: 3cm;
+       height: 3cm;
+       border: 1px solid black;
+       padding: 0.5em;
+       margin: 1em 0;
+    }
+    .bar {
+       background-color: black;    
+       color: white;
+    }
+    .foo div:not(.bar) {
```

Comment:
Changing this selector to an invalid one `.foo div:not(.bar +)`, the rule still applies.
It should be treated as an error and ignored.